hack/os-packages/Makefile

```diff
@@ -31,34 +34,40 @@ download-rpms:
 
 ifeq ($(fips), 1)
 artifacts_dir = $(shell pwd)/artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_el$(rhel_major_version)_x86_64_fips_rpms
-rpm_getter_image = $(RPM_PACKAGES_GETTER_IMAGE)-fips-$(rhel_major_version)
-getter_target = rpm-fips-artifacts-getter
+rpm_getter_image = $(RPM_PACKAGES_GETTER_IMAGE)-fips-centos:$(rhel_major_version)
```

Comment:
The original code was using k8s version as tag for the rpm-getter image. IMHO since multiple k8s versions can be built against same base OS image, I changed the tag to be `major_version` of the OS.

Reply:
This change makes sense!